src/components/carousel/NavigationButton.tsx

```diff
@@ -8,41 +8,28 @@ export type Direction = 'previous' | 'next';
 interface Props {
   onClick: (direction: Direction) => void;
   direction: Direction;
+  fullScreen: boolean;
 }
 
-const NavigationButton: FC<Props> = ({ direction, onClick }) => {
+const NavigationButton: FC<Props> = ({ direction, onClick, fullScreen }) => {
+  const { button, buttonContainer, icon } = useMultiStyleConfig(
+    'Carousel',
+    fullScreen ? { variant: 'fullScreen' } : {}
+  );
   const side = direction === 'previous' ? { left: '0' } : { right: '0' };
 
   return (
     <chakra.button
       onClick={() => onClick(direction)}
-      position="absolute"
-      top="0"
       {...side}
-      width="lg"
-      height="full"
-      display="flex"
-      justifyContent="center"
-      alignItems="center"
-      color="white"
-      visibility="hidden"
-      _groupHover={{ md: { visibility: 'visible' } }}
       aria-label={`${direction} slide`}
+      __css={buttonContainer}
     >
-      <Flex
-        justifyContent="center"
-        alignItems="center"
-        borderRadius="sm"
-        width="md"
-        height="md"
-        backgroundColor="gray.900"
-        opacity="40%"
-        _hover={{ opacity: '80%' }}
-      >
+      <Flex __css={button}>
         {direction === 'previous' ? (
-          <ChevronLeftLargeIcon color="white" />
+          <ChevronLeftLargeIcon boxSize={undefined} __css={icon} />
```

Comment:
We define a default `boxSize` prop for generated `Icons`. Since we want to pass it via chakra theme we need to set it to `undefined` so that the theme value is not overwrittten